### PR TITLE
Key rollover okta 131959

### PR DESCRIPTION
**Pull request by bellatumaneng-okta**
## Description:
- Original doc changes for key rollover in apps API had `autoKeyRollover` in snake case instead of camel case.
- Updates examples and descriptions.

### Resolves:
* [Issue #131959](#131959)

* [OKTA-131959](https://oktainc.atlassian.net/browse/OKTA-131959)

### CODE REVIEW

**Comment by mystiberry-okta** [comment]
Don't merge until after thursday 5pm. We have a release going out tomorrow.

File: _source/_code/php/quickstart-signin-widget.md [hunk 1]
```diff
@@ -1,6 +1,6 @@
 ---
 layout: software
-title: PHP
+title: Okta Sign-In Widget
```
Comment (jmelberg-okta) [inline _source/_code/php/quickstart-signin-widget.md]: Any reason we updated the title in this PR?

Comment (mystiberry-okta) [reply _source/_code/php/quickstart-signin-widget.md]: I didn't update it on purpose, it was a conflict that I resolved. If that's the wrong title, just tell me the right one and I'll fix it.

**Comment by mystiberry-okta** [comment]
@bellatumaneng-okta It looks like you still need to rebase and get rid of all the extra commits. My comment to Jordan is just to let him know I saw the issue for my commit.

**Comment by bellatumaneng-okta** [comment]
@mystiberry-okta it seems that I'm already up to date with the weekly branch. Which extra commits are you referring to?

**Comment by mystiberry-okta** [comment]
@bellatumaneng-okta , If you go here: https://github.com/okta/okta.github.io/pull/1237/commits

you'll see commits from 9/6 and 9/7 that shouldn't be in this PR (it's because of the way we handle release branch, long story). Notice this is based on 'weekly', not 'master.'  

A process improvement is on the way so you shouldn't have to do all this rebasing and wrassling with Gemfile.lock in the future.

**Comment by mystiberry-okta** [comment]
Also you can resolve Gemfile.lock to match HEAD and it'll be okay. In near future this won't happen, I'm told.

**Comment by bellatumaneng-okta** [comment]
@mystiberry-okta understood! I just cleaned it all up.

**Pull request merged**